javascript-loops-practice/get-student-names.js

```diff
@@ -0,0 +1,10 @@
+/* exported getStudentNames */
+function getStudentNames(students) {
+  const names = [];
+  for (const key in students) {
+    names.push(students[key].name);
+    console.log(names);
+  }
```

Comment:
`for..in` loops are for objects. You don't want to use `for..in` loops with arrays. They work, because arrays are technically objects, but you should be using a `for` loop instead